src/PresentationalComponents/Cards/OverviewDashbarCard/OverviewDashbarCard.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import propTypes from "prop-types";
+
+import { Card, CardBody, Flex, FlexItem, Text } from "@patternfly/react-core";
+import { DashbarCardTitle } from "./components/DashbarCardTitle/DashbarCardTitle";
+import { DashbarCardTagOrIcon } from "./components/DashbarCardTagOrIcon/DashbarCardTagOrIcon";
+import {
+  PATHWAYS,
+  INCIDENTS,
+  IMPORTANT_RECOMMENDATIONS,
+  CRITICAL_RECOMMENDATIONS,
+} from "../../../AppConstants";
+
+// this component returns the appropriate Card component based on the title it receives
+export const OverviewDashbarCard = ({ title, count, onClickFilterByTitle }) => {
+  const isWrongTitle =
```

Comment:
This is a bit odd... the const is named `isWrongTitle` and on line 23 you are negating it, so it becomes "isRightTitle"... Maybe there is a better way to name this. 

Also, instead of multiple `... != ... && ... && ...` you can put the consts like `PATHWAYS`, `INCIDENTS`, etc. into an array and use `.includes` instead.

Reply:
Sorry for the confusion 😅 
What you have suggested is a much more simple and readable method, I have added it to the code :) 
Thank you ! @bastilian 

This is what I wrote:
```
export const OverviewDashbarCard = ({ title, count, onClickFilterByTitle }) => {

  const supportedTitles = [PATHWAYS, INCIDENTS, IMPORTANT_RECOMMENDATIONS, CRITICAL_RECOMMENDATIONS];
  
  if(supportedTitles.includes(title))
    return (
      <Card isFullHeight className="dashbar-item">
        <CardBody>
          <DashbarCardTitle title={title} />
  
          <Flex spaceItems={{ default: "spaceItemsSm" }}>
            <FlexItem>
              <DashbarCardTagOrIcon title={title} />
            </FlexItem>
            <FlexItem>
              <Text className="pf-u-font-size-lg pf-u-font-weight-bold pf-u-mt-xs">
                <a
                  onClick={() => onClickFilterByTitle(title)}
                  data-testid={title}
                >
                  {count}
                </a>
              </Text>
            </FlexItem>
          </Flex>
        </CardBody>
      </Card>
    )
  
  return <></>;
};
```